### ISSUE TITLE
new article on site

### ISSUE THREAD

**Issue by alexfrieden** [infinitesunrise/carsinbikelanes#56]
Thought you would want to know, made it clear that you had written it and we were working together to get the boston site up. 

http://www.bostonglobe.com/metro/2016/11/30/interactive-map-let-cyclists-report-drivers-parked-bike-lanes/j9HYFCS2mUHsypWgckvAfO/story.html

carsinbikelanesboston.com

**Comment by AdeasTridae** [comment]
Alex - glad to see this happen. Was thinking of setting this up myself.

BTW - since there's no contact info on the boston site, I can't report a bug to you. On both my iPhone and via Chrome I get an error submitting an image. Error message attached.
![error](https://cloud.githubusercontent.com/assets/23641516/20764467/4aeed49c-b6fc-11e6-822e-3faf40b793a3.jpg)

Here's the image file in question.
![fullsizerender](https://cloud.githubusercontent.com/assets/23641516/20764488/67c2fd6e-b6fc-11e6-80e9-1749c69ff526.jpg)

**Comment by alexfrieden** [comment]
It appears your file is 0 in size.  Let me try uploading.

**Comment by alexfrieden** [comment]
Interesting, appears File name appears to be empty.  

> [Wed Nov 30 17:54:50.211743 2016] [:error] [pid 22509] [client 199.167.124.185:49868] PHP Warning:  getimagesize(): Filename cannot be empty in /var/www/html/submission.php on line 23, referer: http://carsinbikelanesboston.com/index.php?center=42.35,-71&zoom=12
[Wed Nov 30 18:16:32.056487 2016] [:error] [pid 22744] [client 173.252.120.123:15114] PHP Notice:  Undefined offset: 1 in /var/www/html/index.php on line 100
[Wed Nov 30 18:18:23.858554 2016] [:error] [pid 22996] [client 170.223.207.86:13116] submission.php - $date_occurrence inbound: 2016-08-04T08:32:00, referer: http://carsinbikelanesboston.com/index.php?center=42.35,-71&zoom=12
[Wed Nov 30 18:18:23.858580 2016] [:error] [pid 22996] [client 170.223.207.86:13116] submission.php - $date_added inbound: 2016-11-30T18:18:23+00:00, referer: http://carsinbikelanesboston.com/index.php?center=42.35,-71&zoom=12
[Wed Nov 30 18:18:23.858597 2016] [:error] [pid 22996] [client 170.223.207.86:13116] PHP Warning:  getimagesize(): Filename cannot be empty in /var/www/html/submission.php on line 23, referer: http://carsinbikelanesboston.com/index.php?center=42.35,-71&zoom=12

I'll restart the server and give it a shot.

**Comment by alexfrieden** [comment]
Still same problem @infinitesunrise any idea why a 2.2 MB file is showing up as 0?

**Comment by AdeasTridae** [comment]
Mmmm, @alexfrieden - I exported a new picture from my iPhone using the "Medium" option to e-mail it to myself and submit through the website via desktop and that went through - you should see it in the queue to be approved. Here's the smaller file for reference.

![fullsizerenderv3](https://cloud.githubusercontent.com/assets/23641516/20768225/1a93c758-b70b-11e6-8b2d-0e8d380e6f77.jpg)

Could it be a file size issue?

**Comment by alexfrieden** [comment]
Should be approved now.  So you needed a smaller file size to successfully submit?

**Comment by AdeasTridae** [comment]
Yeah - submitting it from site via iPhone (off of my photostream) and desktop Chrome (by e-mailing full res imagine to myself) both raised the same error above. However if I e-mailed the photo and when prompted by my iPhone "You can reduce the file size by scaling the image" selecting the "Medium" option, I could then submit that via desktop Chrome.

**Comment by infinitesunrise** [comment]
Alex, I bet the post_max_size variable is either not set or set too low in your php.ini. In both my dev environment and web server I've got it set to post_max_size = 100M.

And congratulations on making the Globe! Awesome to see CIBL making waves in more than one city already. It's terrifying how fast it takes off :P

**Comment by infinitesunrise** [comment]
Sorry, also these two:

upload_max_size = 100M
upload_max_filesize = 20M

I'm not sure which of these is the relevant one.
